app/react/common/state/users/users.reducers.js

```diff
@@ -33,9 +34,8 @@ export default function reducer(state = initialState, action) {
                 isLoading: false
             };
         case TOGGLE_USER: {
-            const userId = action.userId.toString();
```

Comment:
No need to convert to string now since it numbers everywhere now, so removed this